nilearn/plotting/html_document.py

```diff
@@ -155,41 +190,21 @@ def open_in_browser(self, file_name=None, temp_file_lifetime=30):
             HTML file to use as a temporary file.
 
         temp_file_lifetime : :obj:`float`, default=30
-            Time, in seconds, after which the temporary file is removed.
-            If None, it is never removed.
 
+            .. deprecated:: 0.11
 
+                The parameter is kept for backward compatibility and will be
+                removed in a future version. It has no effect.
         """
+        if temp_file_lifetime != "deprecated":
```

Comment:
Forgot about it but maybe better to use this helper decorator for this:

https://github.com/nilearn/nilearn/blob/1374eee6159f048072efa9c9c7db62cbac28f96d/nilearn/_utils/helpers.py#L109